internal/metrics/metrics.go

```diff
@@ -0,0 +1,43 @@
+package metrics
+
+import (
+	"net/http"
+
```

Comment:
how does lint allow this empty line?

Reply:
Format automatically separates builtin from installed packages